packages/dd-trace/test/encode/coverage-ci-visibility.spec.js

```diff
@@ -88,4 +94,20 @@ describe('coverage-ci-visibility', () => {
     expect(decodedCoverages.coverages).to.have.length(1)
     expect(decodedCoverages.coverages[0]).to.contain({ test_session_id: 3, test_suite_id: 4 })
   })
+
+  it('should be able to encode test coverages', () => {
+    encoder.encode(formattedCoverageTest)
+
+    const form = encoder.makePayload()
+
+    expect(form._data[1]).to.contain('Content-Disposition: form-data; name="coverage1"; filename="coverage1.msgpack"')
+    expect(form._data[2]).to.contain('Content-Type: application/msgpack')
+
+    const decodedCoverages = msgpack.decode(form._data[3])
+
+    expect(decodedCoverages.version).to.equal(2)
+    expect(decodedCoverages.coverages).to.have.length(1)
+    expect(decodedCoverages.coverages[0]).to.contain({ test_session_id: 5, test_suite_id: 6, span_id: 7 })
+    expect(decodedCoverages.coverages[0].files[0]).to.eql({ filename: 'file3.js' })
```

Comment:
`equal` does not work because it's an identify check. I could use `.to.contain` which is similar, but it wouldn't fail if `files[0]` adds an extra field, which it's desirable